28-Github Profile/script.js

```diff
@@ -0,0 +1,101 @@
+const ApiUrl = 'https://api.github.com/users/'
+const form = document.getElementById('form')
+const search = document.getElementById('search')
+const main = document.getElementById('main')
+
+
+async function getUser(username) {
+  try {
+    search.disabled = true
+    const { data } = await axios(ApiUrl + username)
+
+    createUserCard(data)
+    getRepos(username)
+  } catch (err) {
+    if (err.response.status == 404) {
+      createErrorCard('No profile with this username')
+    }
+  }
+  search.disabled = false
```

Comment:
this is not a mistake per se, However I think it will be better to wrap this line in a `.finally` block.

for your info, `.finally` runs after the either the `try` or the `catch` block, but it will run either way.